nss-evening-cohort-26/FE-Hackathon-music-streaming-project
# Playlist detail/ym

## Description
I added to the playlist detail page so the songs would show up based on the playlist selected.

**I have not added the playlist name to the detail page. Just making a PR so Brit can start her branch**

## Related Issue
#9 

## Motivation and Context
-This allows us to view the songs in our playlist which we need in order to then add more songs.

## How Can This Be Tested?
-click on the eye icon in playlist card in order to view the playlist detail page. You should see all the songs associated with that playlist.

## Screenshots (if appropriate):

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
